dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/PenetrateAttachmentSelector.java

```diff
@@ -29,6 +29,7 @@ public interface PenetrateAttachmentSelector {
      *
      * @return attachment pass to next hop
      */
-    Map<String, Object> select();
+    Map<String, Object> select(Invocation invocation);
+    Map<String, Object> selectReverse(Invocation invocation);
```

Comment:
Add some desc of this breaking change in `https://dubbo.apache.org/zh/docs3-v2/java-sdk/upgrades-and-compatibility/`